charts/handbrake/2.0.0/questions.yaml

```diff
@@ -0,0 +1,802 @@
+              - variable: datasetName
+                label: "Dataset Name"
+                schema:
+                  type: string
+                  default: "output"
+                  required: true
+                  editable: false
+              - variable: mountPath
+                label: "Mount Path"
+                description: "Path to mount inside the pod"
+                schema:
+                  type: path
+                  required: true
+                  default: "/output"
+                  editable: false
+              - variable: hostPathEnabled
+                label: "host Path Enabled"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: hostPath
+                      label: "Host Path"
+                      schema:
+                        type: hostpath
+                        required: true
+          # Optical Drive
+        - variable: opticaldrive
+          label: "Optical Drive Device"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  required: true
+                  hidden: true
+                  editable: false
+              - variable: setPermissions
+                label: "Automatic Permissions"
+                description: "Automatically set permissions on install"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+              - variable: name
+                label: "Mountpoint Name"
+                schema:
+                  type: string
+                  default: "opticaldrive"
+                  required: true
+                  editable: true
+              - variable: emptyDir
+                label: "emptyDir"
+                schema:
+                  type: boolean
+                  default: false
+                  hidden: true
+                  editable: false
+              - variable: mountPath
+                label: "Path to Optical Drive device inside the pod"
+                description: "Preferably the same as the path to the Optical Drive on the host"
+                schema:
+                  type: path
+                  required: true
+                  default: ""
+                  editable: true
+              - variable: hostPathEnabled
+                label: "host Path Enabled"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
+              - variable: hostPath
+                label: "Path to Optical Drive device on host"
+                description: 'Best to use path "by-id"'
+                schema:
+                  type: path
+                  required: true
```

Comment:
```suggestion
          # Optical Drive
        - variable: opticaldrive
          label: "Optical Drive Device"
          schema:
            type: dict
            attrs:
              - variable: enabled
                label: "Optical Drive Device"
                schema:
                  type: boolean
                  default: false
                  show_subquestions_if: true
                  subquestions:
                    - variable: setPermissions
                      label: "Automatic Permissions"
                      description: "Automatically set permissions on install"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                    - variable: name
                      label: "Mountpoint Name"
                      schema:
                        type: string
                        default: "opticaldrive"
                        required: true
                        editable: true
                    - variable: emptyDir
                      label: "emptyDir"
                      schema:
                        type: boolean
                        default: false
                        hidden: true
                        editable: false
                    - variable: mountPath
                      label: "Path to mount device inside the container"
                      description: "Preferably the same as the path to the device path on the host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/sr0"
                        editable: true
                    - variable: hostPathEnabled
                      label: "host Path Enabled"
                      schema:
                        type: boolean
                        default: true
                        hidden: true
                    - variable: hostPath
                      label: "Path to Optical Drive device on host"
                      schema:
                        type: path
                        required: true
                        default: "/dev/sr0"
```

I'm changing it to this, (Copy paste from the home-assistant usb) and changed labels
But when i'm installing without checking the checkbox i get this error
```
Error: [EFAULT] Failed to install chart release: Error: template: handbrake/templates/common.yaml:1:3: executing "handbrake/templates/common.yaml" at < include "common.all" . >: error calling include: template: handbrake/charts/common/templates/_all.tpl:35:8: executing "common.all" at : error calling include: template: handbrake/charts/common/templates/_deployment.tpl:55:10: executing "common.deployment" at : error calling include: template: handbrake/charts/common/templates/lib/controller/_pod.tpl:76:10: executing "common.controller.pod" at : error calling include: template: handbrake/charts/common/templates/lib/controller/_volumes.tpl:47:3: executing "common.controller.volumes" at : error calling include: template: handbrake/charts/common/templates/lib/storage/_appStorage.tpl:55:4: executing "common.storage.allAppVolumes" at : error calling include: template: handbrake/charts/common/templates/lib/storage/_appStorage.tpl:38:13: executing "common.storage.configureAppVolumes" at : error calling include: template: handbrake/charts/common/templates/lib/storage/_ixVolumes.tpl:7:7: executing "common.storage.retrieveHostPathFromiXVolume" at : error calling eq: incompatible types for comparison
```

Just tried to install homeassistant withut checking the boxes and getting simillar error. So maybe something is wrong. 
either with the questions. Or with the common?

Reply:
Maybe not copy from untested charts next time. ;-)

Anyway: Ill look at this tomorrow :)

Reply:
@stavros-k Last push should solve your issues with Handbrake Optical drive checkbox.